packages/element/package-lock.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
This file should be removed. We don't want to maintain package lock files for individual packages. It's a bug with Lerna that it doesn't respect `.npmrc` file ... I observe that it happens where you run `npm install` when there are 2 different versions of the package in two `package.json` files.